CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,10 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+
+== 2023-05-16
+<p>The Map View has been re-written in JavaFX. This has most of the features of the old one an dcan be found under "Experimental->Views"</p>
```

Comment:
A few things here:
1) I would avoid using the term "JavaFX" here given as the average user may not understand what that means. Perhaps instead refer to it looking more like other views.
2) "an dcan" -> "and can"
3) instead of "Experimental->Views", use Experimental -&gt Views (no quotes)